tests/application/static_site/test_views.py

```diff
@@ -258,13 +258,13 @@ def test_view_export_page(test_app_client, logged_in_rdu_user):
     assert metadata.find("div", attrs={"id": "lowest-level-of-geography-value"}).text.strip() == "UK"
     assert metadata.find("div", attrs={"id": "time-period-value"}).text.strip() == "4 months"
 
-    things_to_know = page.find("div", attrs={"id": "things-you-need-to-know"})
+    things_to_know = page.select_one("#things-you-need-to-know")
```

Comment:
I changed this by accident (thought it was referring to the details component at first). Could revert it, although I think the `select_one()` method is easier to understand (and doesn't rely upon the element type).

Reply:
Seems fine to me.

Reply:
Yeah this seems good to me.